### DISCUSSION TITLE
K-fold cross-validation

### DISCUSSION THREAD

**Discussion by jaivardhankapoor** [Lightning-AI/pytorch-lightning#5820] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I could not find any ways to do k-fold cross-validation. Do we have to put an outer loop over trainer with different dataset splits for manual k-fold cross-validation?

**Comment by williamFalcon** [comment]
@jaivardhankapoor you would have to handle this in the dataloader or add an outer loop.
